acceptance/tests/windows/QA-563_windows_exit_mcollective.rb

```diff
@@ -53,7 +53,7 @@
 #Shutdown MCollective Service on Windows agent and make sure it successfully exits
 agents.each do |agent|
   on agent, 'net stop mcollective' do |result|
-    assert_match(/The Marionette Collective Server service was stopped successfully/, result.stdout, "Failed to stop MCollective service")
+    on agent, 'net stop mcollective'
```

Comment:
This leaves two invocations of `on agent, 'net stop mcollective'`, the second inside the block for the first. Did you intend to replace the entire 3-line block with a single invocation (a la the change below)?